docs/building_website.md

```diff
@@ -0,0 +1,41 @@
+# Build and Test the Website
+
+You may have noticed that the `markdown` files in the `/docs` directory are also displayed on [Kilo's website](https://kilo.squat.ai/).
+If you want to add documentation to Kilo, you can start a local webserver to check out how the website would look like.
+
+## Requirements
+
+Install [yarn](https://yarnpkg.com/getting-started/install).
+
+## Build and Run
+
+To generate the markdown files in `/website/docs` from `/docs` and the headers in `/website/docs`, run:
+```shell
+make website/docs/README.md
+```
+
+Install `node_modules` and build the website's static content with:
+```shell
+make website/build/index.html
+```
+
+Now run:
+```shell
+cd website
+yarn start
+```
+and point your browser to `localhost:3000`.
+
+If you make changes to any of the markdowns in `/docs` and want to reload the local `node` server, run:
+```shell
+make website/docs/README.md -B
+```
+
+You can execute the above while the node server is running.
+It will reload automatically.
+
+## Add a New File to the Docs
+
+If you add a new file to the `/docs` directory, you also need to create a new header file containing the front-matter in `/website/docs/` and rerun the above command.
```

Comment:
some style suggestion and also we should explicitly show the command since we are crossing the boundaries of a section:
```suggestion
If you add a new file to the `/docs` directory, you also need to create a corresponding header file containing the front-matter in `/website/docs/`.
Then, regenerate the markdown for the website with the command:
```shell
make website/docs/README.md
```
```